tech/languages/haskell/haskell-installation.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Haskell
+page: haskell
+section: tech-languages
+order: 1
+version: 7.8.4
+description: >
+  An advanced, general-purpose, purely-functional programming language with
+  non-strict semantics and strong static typing.
+---
+
+# GHC installation
+
+The Glasgow Haskell Compiler (GHC) is an open source native code compiler for
+Haskell.
+
+To install GHC on Fedora, run:
+
+```
+$ sudo dnf install ghc
+```
+
+This will install ghc including the `ghci` read-eval-print-loop, the `haddock`
```

Comment:
First you write GHC in capitals, here not. Why? if you are writing it as a command, please use `ghc`.